public/app/features/plugins/pluginSettings.ts

```diff
@@ -13,7 +13,7 @@ export function getPluginSettings(pluginId: string, options?: Partial<BackendSrv
     return Promise.resolve(v);
   }
   return getBackendSrv()
-    .get(`/api/plugins/${pluginId}/settings`, undefined, undefined, options)
+    .get(`/api/plugins/${pluginId}/settings`, undefined, undefined, { sanitizePath: true, ...options })
```

Comment:
Just to make sure, we want consumers of `getPluginSettings` to be able to override the `sanitizePath` prop here and that is a intentional design? Otherwise we should flip the args
```suggestion
    .get(`/api/plugins/${pluginId}/settings`, undefined, undefined, { ...options, sanitizePath: true })
```

Reply:
Good catch; we do not want that to try our best to be consistent .